modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -12,13 +12,14 @@ You can customize a developer catalog by using the form view in the Web Console.
 .Prerequisites
 
 * An OpenShift web console session with cluster administrator privileges.
+* The Developer tab is enabled.
```

Comment:
The tab is shown if the perspective is not disabled, so you might say maybe this here instead? Wdyt?

```suggestion
* The Developer perspective is enabled.
```

Reply:
Hi @jerolimov! Thanks for the review. I used tab as in the epic exploration miro, (https://miro.com/app/board/uXjVOoSAPbA=/?moveToWidget=3458764537705006956&cot=14) it was mentioned as "tab" instead of "perspective". The 5th point stated in the exploration is: "AC 5 Disable the Developer tab when Developer tab is disabled". Do let me know if you still want me to use perspective instead of tab.